main_bench_test.go

```diff
@@ -31,7 +30,7 @@ func BenchmarkMergeSort(b *testing.B) {
 		_ = err
 	}
 	f.Close()
-	dir, err := ioutil.ReadDir("testdata/chunks")
+	dir, err := os.ReadDir("testdata/chunks")
```

Comment:
Just curious, I did not check. Do you know the difference btw the two? I guess it's also one less package to import

Reply:
Since `v1.17` they have moved a few functions around and announced the deprecation on the old ones.